internal/providers/qemu/qemu.go

```diff
@@ -38,12 +41,22 @@ func FetchConfig(f *resource.Fetcher) (types.Config, report.Report, error) {
 		return types.Config{}, report.Report{}, err
 	}
 
-	data, err := ioutil.ReadFile(firmwareConfigPath)
-	if os.IsNotExist(err) {
```

Comment:
I realize that the code is changed in a way we don't want. If we first have a read error and then the test the other path were the file is not existing, we don't have an error at all which is not what we want.

So I suggest to this
```
data := []byte{}
for _, path := range firmwareConfigPaths {
	data, err = ioutil.ReadFile(path)
	if os.IsNotExist(err) {
		f.Logger.Info("QEMU firmware config was not found. Ignoring...")
	} else if err != nil {
		f.Logger.Err("couldn't read QEMU firmware config: %v", err)
		return types.Config{}, report.Report{}, err
	}
}
```
The only change to the original behavior is to embed it in a loop.

Reply:
Changed as suggested, except for a minor change in an error message.

Reply:
Ah, yes, the path is good in the error message!